### ISSUE TITLE
Tooltips Insanely Big

### ISSUE THREAD

**Issue by phendrome** [RagedUnicorn/wow-gearmenu#9]
Nice updating to 1.1 -- lots of improvements. Great job on that. No longer the cooldown timer issues -- if you had OmniCC you'd have duplicate cooldown timers (same with CCT which I tried too). 

**Only thing now, is that the tooltips are insanely big now**. Is it possible to use the in-built tooltip instead of using your own? These are waaaay off right now. Even with the smaller option they are still as huge with the same positioning.

**Comment by RagedUnicorn** [comment]
Hi thanks :)

Can you attach a screenshot of the tooltip? I'm not sure if there is a bug that makes the tooltips too big for you or if we just have a different understanding of size. I'm not doing much custom regarding the tooltips and I would consider them normal size. Not sure though if you encountered a bug however.

Regards,
RagedUnicorn

**Comment by phendrome** [comment]
I'll show ya. I'll attach two screenshots. One with a normal "mouseover" and the other one when I'm using GearMenu.

![normal](https://user-images.githubusercontent.com/2565750/47959264-b7d9b780-dfdf-11e8-91f3-5f74a9759a67.jpg)
![gearmenu](https://user-images.githubusercontent.com/2565750/47959265-bad4a800-dfdf-11e8-8215-31a4ad33595d.jpg)

As you can see. There's a big difference. Worth noting is that I'm using TinyTip as well.

P.S. A little side note, as a pixel-perfect guy, I'd _love_ if the icons would follow the _exact_ same structure as the other icons on the Blizzard UI (if you see how I've set it up on my side there on, you'll probably notice a difference between the size and margin on the icons).

**Comment by RagedUnicorn** [comment]
Hi Thanks for the screenshots. I investigated this a little bit but couldn't really find an issue with the tooltips. While its true that GearMenu is using a custom Tooltip the tooltip inherits from the default one and as of this also has the same design and font sizes.

Blizzard frames:
![wowscrnshot_110418_114740](https://user-images.githubusercontent.com/26184132/47963160-64d82280-e028-11e8-9979-1d38f1395af6.jpg)

Custom frames:
![wowscrnshot_110418_114732](https://user-images.githubusercontent.com/26184132/47965629-b513ac80-e049-11e8-8517-912cfa6a12c2.jpg)

I guessing the problem here is more that Tinytip is customizing the default tooltip to a custom size. Because GearMenu is using a custom one that one is not getting changed.

I agree with you that there is no reason as to why GearMenu should have a custom tooltip. I changed this in commit a35a833d765f60472769c20b03fb3adc37e86c32. So GearMenu is using now the default tooltip by Blizzard.

This needs some more testing. If you have the time feel free to download the master branch and give it a try. I will make sure then to include it in the next release.

I will also see if I can fix the itemslots to have the same design as the normal blizzard slots when I find the time.

Regards,
RagedUnicorn
